InvokePesterStub.ps1

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env pwsh
+
+<#
+.SYNOPSIS
+    Stub around Invoke-Pester command used by VSCode PowerShell extension.
+.DESCRIPTION
+    The stub checks the version of Pester and if >= 4.6.0, invokes Pester
+    using the LineNumber parameter (if specified). Otherwise, it invokes
+    using the TestName parameter (if specified). If the All parameter
+    is specified, then all the tests are invoked in the specifed file.
+    Finally, if none of these three parameters are specified, all tests
+    are invoked and a warning is issued indicating what the user can do
+    to allow invocation of individual Describe blocks.
+.EXAMPLE
+    PS C:\> .\InvokePesterStub.ps1 ~\project\test\foo.tests.ps1 -LineNumber 14
+    Invokes a specific test by line number in the specified file.
+.EXAMPLE
+    PS C:\> .\InvokePesterStub.ps1 ~\project\test\foo.tests.ps1 -TestName 'Foo Tests'
+    Invokes a specific test by test name in the specified file.
+.EXAMPLE
+    PS C:\> .\InvokePesterStub.ps1 ~\project\test\foo.tests.ps1 -All
+    Invokes all tests in the specified file.
+.INPUTS
+    None
+.OUTPUTS
+    None
+#>
+param(
+    # Specifies the path to the test script.
+    [Parameter(Position=0, Mandatory)]
+    [ValidateNotNullOrEmpty()]
+    [string]
+    $ScriptPath,
+
+    # Specifies the name of the test taken from the Describe block's name.
+    [Parameter()]
+    [string]
+    $TestName,
+
+    # Specifies the starting line number of the DescribeBlock.  This feature requires
+    # Pester 4.6.0 or higher.
+    [Parameter()]
+    [ValidatePattern('\d*')]
+    [string]
+    $LineNumber,
+
+    # If specified, executes all the tests in the specified test script.
+    [Parameter()]
+    [switch]
+    $All
+)
+
+$pesterModule = Microsoft.PowerShell.Core\Get-Module Pester
```

Comment:
I thought we concluded in the original PSES PR that we should rather use Get-Command Invoke-Pester to determine the currently loaded Pester version?

Reply:
I think "timewise" it is a wash.  However, by checking if the module is loaded (very fast) and then notifying the user that we are `Importing Pester module`, the user gets some feedback the first time the module loaded.  If you use `Get-Command` to start with, then if the module is not loaded, it will be loaded. And for this module, that takes almost 3 seconds on my laptop.